src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -798,9 +795,9 @@
 
     /**
      * This option creates documentation with the appearance and functionality of documentation generated by
-     * Javadoc 1.1.
+     * Javadoc 1.1. This is no longer supported since Javadoc 1.4 (shipped with JDK 1.4)
      * <br/>
-     * See <a href="https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#1.1">1.1</a>.
+     * See <a href="https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#a1.1">1.1</a>.
      * <br/>
      */
     @Parameter( property = "old", defaultValue = "false" )
```

Comment:
I think this parameter should be deprecated for removal with a seperate ticket.

Reply:
Done in https://issues.apache.org/jira/browse/MJAVADOC-730.

Reply:
Fantastic